README.md

```diff
@@ -255,7 +255,6 @@ Symbols can be used interchangably with strings for metric names.
 `librato-rails` also has special helpers which are available inside your controllers:
 
 #### instrument_action
-_experimental_, this interface may change:
```

Comment:
I think we may want to leave this in there, although this has been around a while I still think we may change this interface some in future versions.